tests/karma/search/containers/TestAddonPage.js

```diff
@@ -19,20 +47,150 @@ describe('AddonPage', () => {
+    });
+
+    const info = Array.from(root.querySelector('.addon--info').childNodes);
+    it('renders the addon info', () => {
+      const infoText = info.map((infum) => infum.textContent);
+      assert.deepEqual(
+        infoText,
+        ['Extension', 'Fully Reviewed', 'View on site', 'Edit on site', 'View homepage',
+         'Email support', 'View support site']);
+    });
+
+    it('renders the AMO page as a link', () => {
+      const url = info.find(
+        (infum) => infum.textContent === 'View on site').firstChild;
+      assert.equal(url.tagName, 'A');
+      assert.equal(url.getAttribute('href'), 'https://addons.mozilla.org/firefox/addon/my-addon/');
+    });
+
+    it('renders the manage page as a link', () => {
+      const url = info.find(
+        (infum) => infum.textContent === 'Edit on site').firstChild;
+      assert.equal(url.tagName, 'A');
+      assert.equal(
+        url.getAttribute('href'), 'https://addons.mozilla.org/developers/addon/my-addon/edit');
+    });
+
+    it('renders the support email as a mailto', () => {
+      const email = info.find((infum) => infum.textContent === 'Email support').firstChild;
+      assert.equal(email.tagName, 'A');
+      assert.equal(email.getAttribute('href'), 'mailto:my-addon@example.com');
+    });
+
+    it('renders the support url as a link', () => {
+      const url = info.find(
+        (infum) => infum.textContent === 'View support site').firstChild;
+      assert.equal(url.tagName, 'A');
+      assert.equal(url.getAttribute('href'), 'https://example.com/my-addon/support');
+    });
+
+    it('renders the homepage as a link', () => {
+      const url = info.find(
+        (infum) => infum.textContent === 'View homepage').firstChild;
+      assert.equal(url.tagName, 'A');
+      assert.equal(url.getAttribute('href'), 'https://example.com/my-addon');
+    });
+
+    it('renders the current version header', () => {
+      assert.equal(
+        root.querySelector('.addon--current-version h2').textContent,
+        'Current version');
+    });
+
+    it('renders the current version', () => {
+      const version = Array
+        .from(root.querySelector('.addon--version-info').childNodes)
+        .map((infum) => infum.textContent);
+      assert.deepEqual(version, ['2.5-beta.1']);
+    });
+
+    it('renders the file info', () => {
+      const file = Array
+        .from(root.querySelector('.addon--file-info').childNodes)
+        .map((infum) => infum.textContent);
+      assert.deepEqual(
+        file, ['Linux', 'Fully Reviewed', '556677 bytes', '2016-04-01T12:11:10', 'Download']);
+    });
   });
 
-  it('renders the slug', () => {
-    const root = render({state: initialState, props: {params: {slug: 'my-addon'}}});
-    assert.equal(root.querySelector('p').textContent, 'This is the page for my-addon.');
+  describe('optional fields', () => {
+    const updatedState = {
+      addons: {
+        'my-addon': Object.assign({}, initialState.addons['my-addon'], {
```

Comment:
I think it would be easier for future maintenance if you did the inverse: rename this `requiredFields` and populate `initialState` up above with it like: 
````
const initialState = {
  addons: {
    'my-addon': {
      ...requiredAddonFields, 
      current_version: '0.1',
      homepage: 'http...',
    },
  },
}
````